doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -52,7 +52,7 @@ Table 4-1 lists the Linux kernel versions that comply with this Reference Archit
 > * Which optional features are used and which optional API-s are available
 > * Which [alfa or beta features](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) are used
 
-In alignment with the [Kubernetes version support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions), a Reference Implementation must use one of three latest minor versions (`n-2`) - e.g. if the latest version is 1.17 then the RI must use either 1.17, 1.16 or 1.15. The Kubernetes distribution or product that is used in the RI must be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) and marked (`X`) as conformant for the Kubernetes version that is being used.
+In alignment with the [Kubernetes version support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions), a Reference Implementation must use one of three latest minor versions (`n-2`) - e.g. if the latest version is 1.17 then the RI must use either 1.17, 1.16 or 1.15. The Kubernetes distribution, product, or deployment project used in the RI must either be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) directly or utilize an installer listed in the aforementioned document (e.g., Kubeadm, Kubespray) and marked (`X`) as conformant for the Kubernetes version that is being used.
```

Comment:
> The Kubernetes distribution, product, or deployment project used in the RI must either be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) directly or utilize an installer listed in the aforementioned document (e.g., Kubeadm, Kubespray) and marked (`X`) as conformant for the Kubernetes version that is being used.

I find this a little confusing, as per my comments in the Issue you raised.  How would the following work for you?

> The Kubernetes distribution, product, or installer used in the RI must be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) and marked (`X`) as conformant for the Kubernetes version that is being used.

Your example of CNF testbed is covered (the installer is Kubespray) - does that work?

Reply:
It may depend on how you read the original text. If read strictly, it says that any distribution or product must be listed. However, projects like metal3 or the CNF testbed are not distributions or products in my understanding. Moreover, they are not listed. Instead they make use of an _installer_ which itself is listed. It is really just this differentiation, I'd like propose.

I am fine with your proposed text (adding just installer) - mine was maybe overly complicated.

Reply:
In addition, we anyway need to run the conformance test suite on RI2 to ensure that whatever gets installed is conformant.

Reply:
@gkunz Already covered by RA-2 requirement [`req.gen.cnt.04`](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter02.md#231-general-requirements)

Reply:
Hi @deebhatia . Right, this is the overall target we are striving for and what matters in the end. Because of that, I just felt that the particular sentence I was referring to was potentially restricting our choices regarding the tooling for RI2 by explicitly mentioning installers.